tests/two_d_primitive_tests.h

```diff
@@ -0,0 +1,35 @@
+#ifndef NXDK_PGRAPH_TESTS_2D_PRIMITIVE_TESTS_H
+#define NXDK_PGRAPH_TESTS_2D_PRIMITIVE_TESTS_H
+
+#include <pbkit/pbkit.h>
+
+#include "test_suite.h"
+
+class TestHost;
+
+class TwoDPrimitiveTests : public TestSuite {
+ public:
+  struct TwoDPrimitiveTest {
+    uint32_t two_d_object_color;
```

Comment:
`two_d` is redundant for all of these, and for this one I'd even drop "object", just `color`

Reply:
done